bin/diff_results

```diff
@@ -207,7 +208,15 @@ def any_nss(classifications):
     return classifications['no steady state'] > 0
 
 
-def diff(before_file, after_file, summary_filename):
+def _obfuscate_key(key, diff_vms):
```

Comment:
Can you give an example of the output of a couple of VMs? I'm struggling to visualise what this does exactly.

Reply:
(some discussion on Mattermost on this topic)

Reply:
How about `rewrite_key`?

Reply:
Works for me. Let's comment it to explain what it does (and give an example, because that'll probably make it clearer).